kernel/dune

```diff
@@ -14,15 +14,10 @@
  (targets copcodes.ml)
  (action (with-stdout-to %{targets} (run ./genOpcodeFiles.exe copml))))
 
-(executable
-  (name write_uint63)
-  (modules write_uint63)
-  (libraries unix))
-
 (rule
  (targets uint63.ml)
- (deps (:gen ./write_uint63.exe) uint63_x86.ml uint63_amd64.ml)
- (action (run %{gen})))
+ (deps (:gen-file uint63_%{ocaml-config:architecture}_%{ocaml-config:int_size}.ml))
```

Comment:
While we're at it, why do we depend on `architecture`? Those files don't have any assembly AFAICT

Reply:
No idea, I just followed previous naming.

Reply:
cc @vbgl

Reply:
Indeed, this information seems redundant. What we really care about here is the int size.